src/components/Navigation/Navigation.jsx

```diff
@@ -118,11 +123,27 @@ export default class Navigation extends React.Component {
         inputSelector: '.navigation__search-input'
       });
 
-      window.addEventListener('keyup', e => {
-        if (e.which === 9 && e.target.classList.contains('navigation__search-input')) {
-          this._openSearch();
+      // Keydown so we can "stop" the event if need be. 
+      window.addEventListener("keydown", e => {
```

Comment:
Please apply for single quotes
```suggestion
      window.addEventListener('keydown', e => {
```